### PR TITLE
Labels table and translation

### PR DESCRIPTION
**Pull request by mahbx**
# Changes :gem:
* add labels table in database
* add tests for reading labels from database, tests pass
* add function for translating labels into Norwegian, not tested.

### CODE REVIEW

File: src/test/test_api.py [hunk 1]
```diff
@@ -46,6 +48,7 @@ def test_start_game_correct(client):
     """
     res = client.get("/startGame", data=dict())
     # Ensure that the returned dictionary contains a token
+    print(f"res {res.data}")
```
Comment (joaaax) [inline src/test/test_api.py:51]: Do we need this print?

File: src/webapp/api.py [hunk 2]
```diff
@@ -59,6 +60,9 @@ def start_game():
     game_id = uuid.uuid4().hex
     token = uuid.uuid4().hex
     labels = random.sample(LABELS, k=NUM_GAMES)
+    print("in start game")
```
Comment (joaaax) [inline src/webapp/api.py:63]: Do we need this?

File: src/webapp/api.py [hunk 3]
```diff
@@ -24,18 +24,19 @@
...
 TIME_LIMIT = setup.time_limit
 NUM_GAMES = setup.num_games
 CERTAINTY_TRESHOLD = setup.certainty_threshold
 HIGH_SCORE_LIST_SIZE = setup.top_n
-
+LABELS = setup.LABELS
 app.config.from_object("utilities.setup.Flask_config")
 models.db.init_app(app)
 models.create_tables(app)
+models.seed_labels(app, "./utilities/Dict_eng_to_nor.csv")
```
Comment (joaaax) [inline src/webapp/api.py:39]: Could we use a small D in the filename?

File: src/webapp/models.py [hunk 4]
```diff
@@ -51,7 +65,7 @@ def create_tables(app):
         The tables will be created if they do not already exist.
     """
     with app.app_context():
-        db.drop_all()  # Temporary
+        # db.drop_all()  # Temporary
```
Comment (joaaax) [inline src/webapp/models.py:68]: Is this comment necessary?

File: src/webapp/api.py (hunk 2 above)
Comment (mahbx) [reply src/webapp/api.py:63]: removed

File: src/webapp/api.py (hunk 3 above)
Comment (mahbx) [reply src/webapp/api.py:39]: changed to small d

File: src/test/test_db.py [hunk 5]
```diff
@@ -60,6 +60,16 @@ def test_insert_into_player_in_game():
...
 
 
+def test_insert_into_labels():
+    """
+        Check that records exists in Games table after inserting.
+    """
+    with api.app.app_context():
+        result = models.insert_into_labels(
+            str(uuid.uuid4().hex), str(uuid.uuid4().hex))
+    assert result
```
Comment (andreacx) [inline src/test/test_db.py:70]: Necessary parsing a str to a str? .hex is already a str

File: src/webapp/api.py [hunk 6]
```diff
@@ -24,18 +24,19 @@
 from flask import json
 from flask_sqlalchemy import SQLAlchemy
 from werkzeug import exceptions as excp
+import pdb
```
Comment (andreacx) [inline src/webapp/api.py:27]: pdb?

File: src/webapp/models.py [hunk 7]
```diff
@@ -271,3 +285,70 @@ def drop_table(table):
...
+    try:
+        # read all english labels in database
+        labels = Labels.query.all()
+        english_labels = [str(label.english) for label in labels]
+        print("LABELS:", english_labels)
+        random_list = random.sample(english_labels, n)
+        return random_list
+
+    except AttributeError:
+        print("Could not read " + str(n) + " random rows from Labels table")
```
Comment (andreacx) [inline src/webapp/models.py:341]: print?

File: src/webapp/models.py [hunk 8]
```diff
@@ -271,3 +285,70 @@ def drop_table(table):
...
+        return AttributeError("Could not read Labels table")
```
Comment (andreacx) [inline src/webapp/models.py:342]: raise error

File: src/webapp/models.py [hunk 9]
```diff
@@ -271,3 +285,70 @@ def drop_table(table):
...
+def to_norwegian(english_label):
+    """
+        Reads the labels tabel and return the norwegian translation of the english word
+    """
+    try:
+        norwegian_word = Labels.query.get(english_label)
+        return norwegian_word
+
+    except AttributeError:
+        return AttributeError("Could not find translation in Labels table")
```
Comment (andreacx) [inline src/webapp/models.py:354]: raise error

**Pull request merged**